hyperspy/drawing/utils.py

```diff
@@ -130,7 +130,15 @@ def create_figure(window_title=None,
     fig : plt.figure
 
     """
+    mpl_was_interactive = False
```

Comment:
This doesn't look a good approach to me. It should possible to include the matplotlib figure in a widget output and display this widget.

Reply:
Yes, that makes much more sense.